include/g_compensation/g_compensator_class.h

```diff
@@ -60,6 +60,14 @@ class GCompensator
             buffer_size_ = 50; //default value
             ROS_WARN("Continuing with default buffer_size '50'.");
         };
+
+        bool negate_wrench_flag_;
+        if (!rosparam_shortcuts::get("g_compensator", n_static_params, "negate_wrench", negate_wrench_flag_))
+        {
+            negate_wrench_flag_ = false; //default value
+            ROS_WARN("Continuing with not negating the wrench received per default.");
```

Comment:
I wonder if we really need a warning here? I think this could be a INFO or better DEBUG, what do you think?

Reply:
I would be okay with making it DEBUG.
Would it also make sense to change the logging level at the `gravity_frame` and the `buffer_size`?

Reply:
indeed!! For `gravity_frame`, I'd keep WARN, since it's an important parameter which you should actively think about and set. However, `buffer_size` is just a possible tuning parameter, which should be INFO or DEBUG.

Reply:
Made them INFO in [609ad3c](https://github.com/ipa320/g_compensator/pull/4/commits/609ad3ccae438371f76b7a88fb6e457213a4aeea).